scripts/get-latest-clients.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This script is good, but I think we could also add support in the "run-moonbeam-fork" script the possibility the pass the version or commit instead of the path as argument (same for polkadot) and it would download (if needed) it

Reply:
Great idea, have done first attempt now which will download latest if cannot find binary in location. Will add ability to specify version in a future commit

Reply:
Script will now download version specified as args. Though keep this script around, as it still has utility.